shared/gh/js/controller/gh.visibility.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright 2014 Digital Services, University of Cambridge Licensed
+ * under the Educational Community License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['gh.api.util'], function(utilAPI) {
+
+    /**
+     * Highlight the selected visibility types
+     *
+     * @private
+     */
+    var selectVisibilityType = function() {
+        $('.gh-visibility-label').removeClass('checked');
+        $(this).parents('label').addClass('checked');
+    };
+
+    /**
+     * Render and show the 'visibility' modal dialog
+     *
+     * @private
+     */
+    var showVisibilityModal = function() {
+        // Render the modal
+        utilAPI.renderTemplate($('#gh-visibility-modal-template'), {'data': {
+            // TODO: make this dynamic once the functionality is available in the back-end
+            'visibility': 'draft'
+        }}, $('#gh-visibility-modal-container'));
+        // Show the modal
+        $('#gh-visibility-modal').modal();
+        return false;
```

Comment:
Since we're not opening a new page, can you change it to a button to be semantically correct? The `new module` button is an example of how it should look.